src/classes/HH_Households_TEST.cls

```diff
@@ -635,17 +717,12 @@ public with sharing class HH_Households_TEST {
 
         npo02__Households_Settings__c householdSettingsForTests = UTIL_CustomSettingsFacade.getHouseholdsSettingsForTests(new npo02__Households_Settings__c (npo02__Household_Rules__c = HH_Households.ALL_PROCESSOR));
 
-        npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_and_Orgs_Settings__c (npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR));
+        setOrgContactSettingsToOneToOne();
 
-        String street = '2345 Elm St.';
-        String city = 'Seattle';
-        String state = 'Washington';
-        String zip = '88777';
-        String country = 'United States';
         String lastname = 'SmithForTest99';
 
         List<Contact> cons = new List<contact>();
-        Integer conCount = 200;
+        Integer conCount = 1;
```

Comment:
Can this still be considered `testBulk` if there's only 1 Contact 😉 I also worry that by reducing the number of Contacts, we're changing what code gets covered by the test method (but I haven't confirmed that...)

Reply:
Yep, definitely a leftover from testing/dev.  Thanks for catching!  Didn't mean to make that change permanent.  Fixed!